flight/Modules/Autotune/autotune.c

```diff
@@ -163,6 +228,8 @@ static void UpdateStabilizationDesired(bool doingIdent) {
 	StabilizationDesiredSet(&stabDesired);
 }
 
+#define MAX_PTS_PER_CYCLE 4
```

Comment:
One thing that would be nice, that I didn't do.. would be to quantize the dTs to multiples of the sensor interval.  It would cut down noise.